spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/mapping/SpannerPersistentEntityImpl.java

```diff
@@ -86,6 +89,9 @@
 
 	private String tableName;
 
+	private SpannerReadConverter readConverter = new SpannerReadConverter();
```

Comment:
Could we pipe in the bean for this? It is kind of a pain, but I think necessary since some of these key properties might be types using custom converters that the user set up. 

I took care to pass in the spannerEntityWrtier in the constructor for this reason too.

Reply:
Just auto-wire the `SpannerEntityProcessor` an call `getReadConverter` on it.

Reply:
We can't use autowire because this shouldn't depend on Spring Boot and they made a big point about that. We've worked hard to keep it that way too